.github/workflows/codeql-analysis.yml

```diff
@@ -5,7 +5,10 @@ on:
     branches: [ main ]
   pull_request:
     # The branches below must be a subset of the branches above
-    branches: [ main ]
+    branches: [ main, feature/648-extend-codeql-scope ]
+    paths-ignore:
```

Comment:
Would be a possibility to exclude test code using this "path-ignore"?

Reply:
This path-ignored prevents unnecessary scans if the change contains only these files (https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#avoiding-unnecessary-scans-of-pull-requests)

There is another paths-ignore in the config file but it works only for interpreted languages:
https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#specifying-directories-to-scan

![Screenshot 2022-02-11 at 11 14 39](https://user-images.githubusercontent.com/4217554/153574040-43714831-bae1-42b2-bfcf-60412bcf7856.png)